packages/relay/src/index.ts

```diff
@@ -19,13 +20,10 @@ import type {
   ProtocolCreateOptions,
   SendResult,
 } from "@waku/interfaces";
+import { groupByContentTopic } from "@waku/utils";
+import { DefaultPubSubTopic, RelayCodecs } from "@waku/utils/constants";
```

Comment:
I would expect `RelayCodecs` to be in `@waku/relay` package. because they specify what version of relay this specific package implements.